test/ConstantsDemo.thrift

```diff
@@ -65,7 +65,9 @@ const map<i32,thing> GEN_WHAT = { 35 : { 'hello' : 325, 'goodbye' : 325352 } }
 
 const set<i32> GEN_SET = [ 235, 235, 53235 ]
 
-const uuid GEN_UUID = "00000000-4444-CCCC-ffff-0123456789ab"
+// uuids are accepted with or without curly braces
+const uuid GEN_UUID =  00000000-4444-CCCC-ffff-0123456789ab
+const uuid GEN_GUID = {00112233-4455-6677-8899-aaBBccDDeeFF}
```

Comment:
1. can we force it still to be in string format? (e.g. adding leading and trailing `"` to the ll file)
2. if we decided that we want to support the form with curly braces, then I'll need to add support to that to the go library.
3. will the compiler still take this as a string token?

Reply:
> an we force it still to be in string format? (e.g. adding leading and trailing " to the ll file)
Why?

Reply:
several reasons:
1. This would likely become a very common mistake, that developers would think that they are supposed to write it in this way: `const uuid GEN_UUID = "00000000-4444-CCCC-ffff-0123456789ab"`
2. The editor supports for the new syntax will take time to catch up (see how github is highlighting the syntax right now for an example), which will further make 1 more likely to happen.

Reply:
We can't fly to Mars because we don't have syntax highlighting for it .... :-D

Reply:
Done. That would have saved me some time if we had made this clear from the beginning.

Reply:
Mars is overrated anyways :-D